src/stream.ts

```diff
@@ -3108,27 +3108,53 @@ export class Part extends Stream {
         return tempM.estimateStaffLength();
     }
 
-    systemWidthsAndBreaks(): [number[], number[]] {
-        const measureWidths = this.getMeasureWidths();
+    /**
+     * Calculate system breaks and update measure widths as necessary on
+     * account of the reiteration of clefs and key signatures on subsequent systems.
+     */
+    systemWidthsAndBreaks(setMeasureWidths: boolean = true): [number[], number[]] {
         const maxSystemWidth = this.maxSystemWidth; // cryptic note: 'of course fix!'?
```

Comment:
If this is fixed, we can remove the cryptic note.

Reply:
My guess is that the comment is saying "hey, don't assume I've checked every substream's maxSystemWidth!" Which I'm not handling in this PR. So maybe I should just update the comment to say what I (guess) that it means. There was another one somewhere that said the same thing.

Reply:
No, I see here we are calling the property which does go up and check activeSites, so, indeed, this comment has outlived its usefulness. I went a few years up the blame tree and found nothing of note.